azext_edge/edge/providers/orchestration/resource_map.py

```diff
@@ -134,7 +134,7 @@ def refresh_resource_state(self):
 
         self._cluster_container = refreshed_cluster_container
 
-    def build_tree(self, include_dependencies: bool = False, category_color: str = "cyan") -> Tree:
+    def build_tree(self, include_dependencies: bool = True, category_color: str = "cyan") -> Tree:
```

Comment:
is this a planned change?

Reply:
Technically a bug stemming from changing this argument from `hide_extensions` to `include_dependencies` - since the AIO extension is no longer a dependency, we still want to show it as an extension even if the other dependencies should not be shown - lots of usage of this method exists with default args so this was just a refactoring oopsie